### PR TITLE
PIL-531 PPN history UI updates

### PR DESCRIPTION
**Pull request by graszka22**

### CODE REVIEW

File: src/components/ActivityFeed/ActivityFeed.js [hunk 1]
```diff
@@ -316,6 +318,11 @@ class ActivityFeed extends React.Component<Props, State> {
             activeTab={activeTab || firstTab}
           />
         }
+        {formattedFeedData.length === 2 && (
```
Comment (vyvil) [inline src/components/ActivityFeed/ActivityFeed.js:321]: Hm, why we show Empty State message if feed is not empty (has 2 list items in it)?

Comment (graszka22) [reply src/components/ActivityFeed/ActivityFeed.js:321]: It has always the header and the tabs. Maybe I can add a comment to make it clearer?

**Comment by graszka22** [comment]
@vyvil empty state fixed, no more weird ifs

File: src/components/ActivityFeed/ActivityFeed.js [hunk 2]
```diff
@@ -123,6 +122,7 @@ type Props = {
   tabsComponent?: React.Node,
   headerComponent?: React.Node,
   flatListProps?: FlatList,
+  isPPNView: boolean,
```
Comment (jdnmr) [inline src/components/ActivityFeed/ActivityFeed.js:125]:
```suggestion
  isPPNView?: boolean,
```

File: src/components/ActivityFeed/ActivityFeedItem.js [hunk 3]
```diff
@@ -114,6 +114,7 @@ type Props = {
   isSmartWalletActivated: boolean,
   assetDecimals: number,
   bitcoinAddresses: BitcoinAddress[],
+  isPPNView: boolean,
```
Comment (jdnmr) [inline src/components/ActivityFeed/ActivityFeedItem.js:117]:
```suggestion
  isPPNView?: boolean,
```

**Pull request merged**